libs/core/langchain_core/language_models/chat_models.py

```diff
@@ -632,6 +679,7 @@ def _astream(
     ) -> AsyncIterator[ChatGenerationChunk]:
         raise NotImplementedError()
 
+    @deprecated("0.1.0", alternative="invoke")
```

Comment:
This means deprecated since 0.1.0

Reply:
yea was imagining we wouldn't land this till 0.1.0